Exercises/4-count-types.js

```diff
@@ -1,5 +1,15 @@
 'use strict';
 
-const countTypesInArray = null;
+const countTypesInArray = array => {
+  const obj = {};
+  for (const value of array) {
+    if (typeof(value) in obj) {
```

Comment:
Use `typeof value` instead of `typeof(value)`

Reply:
Thank! Fixed according to comment